### PR TITLE
Fix #4907 - add more ACMG ClinGen SVI links

### LINKED ISSUE #4907: Additional ACMG PVS1 criteria link
**Is your feature request related to a problem in the current program to new available techology or software? Please describe and add links/citations if appropriate.**
We have Tayoun, but ACMG SVI flowchart would also be nice.

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality.

It turns out the SVI PVS1 flowchart is basically included in the Tayoun publication which we already link. I held off on that, but included other guidelines that were discussed in the GMCK-RD meeting, ie for RNA on PVS1/PM6, as well as in trans and *de novo* semi-quantitative approaches (PS2, PM3).

Screenshot 2024-10-08 at 13 02 16

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. look at text and click links

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [x] tests executed by DN

### CODE REVIEW

File: scout/constants/acmg.py [hunk 1]
```diff
@@ -60,22 +60,24 @@
                     (
                         "PS1",
                         {
-                            "short": "Known pathogenic aa (HQ)",
+                            "short": "Known pathogenic aa",
```
Comment (dnil) [inline scout/constants/acmg.py:63]: The meaning of HQ is unclear to me here. Perhaps the intention was to disambiguate it from PM5 or PP5? Regardless, unless you know, let's remove it.

File: scout/constants/acmg.py [hunk 2]
```diff
@@ -60,22 +60,24 @@
...
+                            "short": "<i>De novo</i> (confirmed)",
                             "description": "De novo (both maternity and paternity confirmed) in a patient with the disease and no family history",
+                            "documentation": 'Strength can be modified based on <a href="https://clinicalgenome.org/site/assets/files/3461/svi_proposal_for_de_novo_criteria_v1_1.pdf" target="_blank">SVI <i>de novo</i></a>.',
                         },
                     ),
                     (
                         "PS3",
                         {
-                            "short": "Functional damage (HQ)",
+                            "short": "Functional damage",
```
Comment (dnil) [inline scout/constants/acmg.py:78]: Here it seems somewhat clear the intention was to indicate the requirement in the criterium that "the evidence is strong". Adding strength modification and more flexible Brnich reference here supersedes this.

File: scout/constants/acmg.py [hunk 3]
```diff
@@ -126,15 +129,16 @@
                     (
                         "PM5",
                         {
-                            "short": "Similar to known pathogenic aa (HQ)",
-                            "description": "Novel missense change at an amino acid residue where a different missense change determined to be pathogenic has been seen before,  the amino acids have similar properties and the evidence is strong",
+                            "short": "Similar to known pathogenic aa",
+                            "description": "Novel missense change at an amino acid residue where a different missense change determined to be pathogenic has been seen before, the amino acids have similar properties and the evidence is strong",
```
Comment (dnil) [inline scout/constants/acmg.py:133]: Again, likely "HQ" to mean "evidence is strong". This is somewhat evident from Richards 2015. Let's skip it.

File: scout/constants/acmg.py [hunk 4]
```diff
@@ -147,36 +151,40 @@
                     (
                         "PP1",
                         {
-                            "short": "Cosegregation (WQ)",
```
Comment (dnil) [inline scout/constants/acmg.py:150]: "WQ" is likely for "the evidence is weak". Using Biesecker 2023 for strenght modulation is recommended instead.

File: scout/constants/acmg.py [hunk 5]
```diff
@@ -222,15 +231,17 @@
                     (
                         "BS3",
                         {
-                            "short": "No functional damage (HQ)",
+                            "short": "No functional damage",
```
Comment (dnil) [inline scout/constants/acmg.py:234]: "HQ" for "evidence is strong" - added Brnich 2019 instead.

File: scout/constants/acmg.py [hunk 6]
```diff
@@ -222,15 +231,17 @@
...
+                            "documentation": 'Strength can be modified based on <a href="https://clinicalgenome.org/docs/recommendations-for-application-of-the-functional-evidence-ps3-bs3-criterion-using-the-acmg-amp-sequence-variant-interpretation/" target="_blank">Brnich 2019</a>.',
                         },
                     ),
                     (
                         "BS4",
                         {
-                            "short": "Non-segregation (HQ)",
+                            "short": "Non-segregation",
                             "description": "Lack of segregation in affected members of a family, and the evidence is strong",
+                            "documentation": 'Strength can be modified based on <a href="https://clinicalgenome.org/docs/clingen-guidance-for-use-of-the-pp1-bs4-co-segregation-and-pp4-phenotype-specificity-criteria-for-sequence-variant/" target="_blank">Biesecker et al 2023</a>',
```
Comment (dnil) [inline scout/constants/acmg.py:244]: Same here on HQ - strong evidence; added Biesecker 2023.

**Comment by northwestwitch** [comment]
Nice one! 🚀 I'll review first thing tomorrow

File: scout/constants/acmg.py (hunk 1 above)
Comment (northwestwitch) [reply scout/constants/acmg.py:63]: I have no idea, let's remove it!

File: scout/constants/acmg.py [hunk 7]
```diff
@@ -47,7 +47,7 @@
                         {
                             "short": "Null variant",
                             "description": "Null variant (nonsense, frameshift, canonical +/- 2 bp splice sites, initiation codon, single or multiexon deletion) in a gene where LOF is a known mechanism of disease.",
-                            "documentation": 'Strength can be modified based on <a href="https://pubmed.ncbi.nlm.nih.gov/30192042/" target="_blank">Tayoun et al</a> and <a href="http://autopvs1.genetics.bgi.com/" target="_blank">AutoPVS1</a>.',
+                            "documentation": 'Strength can be modified based on <a href="https://pubmed.ncbi.nlm.nih.gov/30192042/" target="_blank">Tayoun et al</a> and <a href="http://autopvs1.genetics.bgi.com/" target="_blank">AutoPVS1</a>, or for RNA <a href="https://www.clinicalgenome.org/docs/application-of-the-acmg-amp-framework-to-capture-evidence-relevant-to-predicted-and-observed-impact-on-splicing-recommendations/" target="_blank">Walker et al</a>.',
```
Comment (northwestwitch) [inline scout/constants/acmg.py:50]: Ther are these tables that can be opened quickly on this other link..

File: scout/constants/acmg.py [hunk 8]
```diff
@@ -60,22 +60,24 @@
...
                             "description": "Same amino acid change as a previously established pathogenic variant regardless of nucleotide change",
                         },
                     ),
                     (
                         "PS2",
                         {
-                            "short": "De novo (confirmed)",
+                            "short": "<i>De novo</i> (confirmed)",
                             "description": "De novo (both maternity and paternity confirmed) in a patient with the disease and no family history",
+                            "documentation": 'Strength can be modified based on <a href="https://clinicalgenome.org/site/assets/files/3461/svi_proposal_for_de_novo_criteria_v1_1.pdf" target="_blank">SVI <i>de novo</i></a>.',
```
Comment (northwestwitch) [inline scout/constants/acmg.py:72]: 👍🏻

**Review by northwestwitch — APPROVED** [review]
It looks great! I'm sure these new link will be very appreciated by clinicians! 🥇

**Pull request merged**